### PR TITLE
feat: CSV Flatfile Pipeline Testing

### PR DESCRIPTION
**Pull request by rymarczy**
This PR provides a new testing method `test_whole_table`. This method takes a CSV representation of Vehicle Position records, converts it to a Parquet file on the fly and runs it through `process_gtfs_rt_files` to populate GTFS-RT RDS tables. 

After Processing, a query selects records from the RDS and compares them against a CSV results file to confirm that there have been no fundamental changes to the Performance Manager processing pipelines.

The input CSV file consists of mostly whole route trips, 3 in each direction for each MBTA Rail line. 

This PR required the updating of GTFS and GTFS-RT test files in the `SPRINGBOARD` `test_files` folder of the repository to match that date of records from the flat file (May 8th 2023). In an effort to reduce the repository size, all non rail data has also been stripped from the `SPRINGBOARD` parquet files of `RT_TRIP_UPDATES` and `RT_VEHICLE_POSITIONS`. 

This Branch was developed with a commit from prior to the inclusion of the PR that removed hash columns from the application. This was done to provide a check against the validity of the changes introduced by that PR. 

Asana Task: https://app.asana.com/0/1204931901750655/1205084207879142

### CODE REVIEW

File: python_src/tests/performance_manager/test_performance_manager.py [hunk 1]
```diff
@@ -445,15 +458,135 @@ def test_vp_and_tu(
...
+        os.path.join(test_files_dir, "pipeline_flat_out.csv"),
+        dtype=result_dtypes,
+    )
+    csv_result_df = csv_result_df.sort_values(
+        by=sort_by,
+        ignore_index=True,
+    )
+
+    compare_result = db_result_df.compare(csv_result_df, align_axis=1)
+    print(compare_result, flush=True)
```
Comment (mzappitello) [inline python_src/tests/performance_manager/test_performance_manager.py:589]: only print this if the assert fails?

**Review by mzappitello — CHANGES_REQUESTED** [review]
two small change requests, but it looks good to me.

File: python_src/tests/performance_manager/test_performance_manager.py [hunk 2]
```diff
@@ -445,15 +458,135 @@ def test_vp_and_tu(
...
+            ~MetadataLog.path.contains("FEED_INFO")
+        )
+    )
+
+    csv_file = os.path.join(test_files_dir, "vehicle_positions_flat_input.csv")
+    parquet_folder = "RT_VEHICLE_POSITIONS/year=2023/month=5/day=8/hour=11"
+    parquet_file = os.path.join(
+        springboard_dir, parquet_folder, "flat_file.parquet"
+    )
+    os.makedirs(os.path.join(springboard_dir, parquet_folder), exist_ok=True)
```
[the remark this answers is not in the document]
Comment (rymarczy) [reply python_src/tests/performance_manager/test_performance_manager.py]: Incorporated with pytest `temp_path` fixture.

File: python_src/tests/performance_manager/test_performance_manager.py (hunk 1 above)
Comment (rymarczy) [reply python_src/tests/performance_manager/test_performance_manager.py]: Incorporated.

**Review by mzappitello — APPROVED** [review]
LGTM 🍰

**Pull request merged**